### ISSUE TITLE
vue-async-computed doesn't work in IE

### ISSUE THREAD

**Issue by johannes-z** [foxbenjaminfox/vue-async-computed#63]
I cannot use `vue-async-computed` in IE11 using it like this: `import AsyncComputed from 'vue-async-computed'`. I get the follow Error:

```
SCRIPT1028: Expected identifier, string or number
chunk-vendors.js (7,10068)
```

This is the file/line throwing the error:

```js
function makeLazyComputed (key) {
  return {
    get () { // <----------------------------------------
      this[lazyActivePrefix + key] = true;
      return this[lazyDataPrefix + key]
    },
    set (value) {
      this[lazyDataPrefix + key] = value;
    }
  }
}
```

**Comment by foxbenjaminfox** [comment]
This should be fixed in `v3.5.2`, released yesterday. `v3.5.1` had [an issue](https://github.com/foxbenjaminfox/vue-async-computed/issues/62) with exporting in the `module` field of `package.json` a non-transpiled version of the package, and tools such as webpack would pick that up instead of the `main` file.

While you could configure your build pipeline to transpile the package yourself, there is no reason for you to have to do that. `v3.5.2` fixes this issue, and makes it that both the `esm` and the `umd` versions of the package are transpiled with babel.

In short: update the version of `vue-async-computed` that your using, and this problem will go away. If it does not, or you're still having trouble, feel free to reopen this issue.
